pyramid/authentication.py

```diff
@@ -565,8 +565,16 @@ class AuthTktAuthenticationPolicy(CallbackAuthenticationPolicy):
         steps.  The output from debugging is useful for reporting to maillist
         or IRC channels when asking for support.
 
+    ``samesite``
+
+        Default: ``'Lax'``.  The 'samesite' option of the session cookie. Set
+        the value to ``None`` to turn off the samesite option.
+
+        This option is available as of :app:`Pyramid` 1.10.
```

Comment:
Move the version info down to a `.. versionchanged:: 1.10`` directive at the bottom of the docstring and document it as "Added the ``samesite`` option and made the default ``'Lax'``." similar to below in the csrf storage policy.